editor.planx.uk/src/pages/FlowEditor/components/Team/components/UserUpsertModal.tsx

```diff
@@ -136,7 +136,7 @@ export const EditorUpsertModal = ({
         >
           <Box sx={{ mt: 1, mb: 4 }}>
             <Typography variant="h3" component="h2" id="dialog-heading">
-              Add a new editor
+              Add a new { isDemoTeam ? "demo user" : "editor" }
```

Comment:
Should `"editor"` here not be `"member"`?

Reply:
Or is it now that we're adding a user, we can now specify the type of role they can receive?

Reply:
Yep exactly - this is specifically adding a user with the `teamEditor` role, so "editor" is correct.

I think there's certainly scope for a role picker within this modal, but I think this is a separate task so I'm keeping thing simple here!

Reply:
Only grievance I would have with this is that you click "Add a team member" but receive a modal saying "Add a team editor" - feels like there's room for confusion. Could it be worth keeping the modal heading the same but alter the info text of the modal to give guidance on what permissions the team member will have?

This is a small nitty comment, so happy for it to be parked for future